### ISSUE TITLE
Aria-label is being applied to 'div' instead of the 'input'

### ISSUE THREAD

**Issue by lalong13** [gregnb/mui-datatables#1152]
## Expected Behavior
In the table search component, the `input` should have `aria-label="Search"` or equivalent.

## Current Behavior
In the table search component, the `aria-label` is being applied to the `div` that contains the `input` rather than on the `input` itself. 

## Steps to Reproduce (for bugs)

1. Any table with contents and an active search input
2. Inspect the `input` element and its containing `div`

## Your Environment

| Tech         | Version |
|--------------|---------|
| Material-UI  |   3.x |
| MUI-datatables  |   2.13.3  |
| React        |  16.12.0  |
| browser      |   Chrome 79.0.3945.117 |

**Comment by lalong13** [comment]
Proposed fix for this issue here:
https://github.com/gregnb/mui-datatables/pull/1153
